signup.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="utf-8">
+    <title>회원가입</title>
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css">
+    <link rel="stylesheet" href="styleLogin.css">
+  </head>
+  <body>
+    <div class="content">
+      <div class="head-img">
+        <a class="login-link" href="/">
+          <img class="pandaMarketImg" src="img/logo.png">
+        </a>
+      </div>
+      <div>
+        <p>이메일</p> 
+        <input type="email" name="userEmail" id="userName">
+        <p>닉네임</p> 
+        <input type="text" name="userName" id="nicname">
+        <p>비밀번호</p>
+        <input type="password" name="userPassword" id="userPassword">
+        <p>비밀번호 확인</p>
+        <input type="password" name="userPassword" id="userPassword">
+      </div>
+      <div class="content-login">
+        <a class="login-button" href="/signup">회원가입</a>
+      </div>
```

Comment:
❗️ 수정요청
해당 태그는 회원가입 동작을 수행하는 태그이므로 button 태그가 더 적절합니다.
즉 form에서 제출 이벤트가 발생했을때 해당 form 안의 유저가 입력한 정보를 제출하는 역할을 하므로
button으로 작성해주세요!

```suggestion
      <div class="content-login">
        <button type="submit" class="login-button">회원가입</button>
      </div>
```

Reply:
❗️ 수정요청  
위에서 언급드린 button 동작의 이점을 가지고 가기 위해서는 form 태그를 사용하시는 것이 좋습니다!  
회원가입에서 form 영역은 회원가입할때 필요한 데이터들의 구역을 의미합니다.  
간편 로그인하기 부분은 회원가입시 필요한 정보가 아니니 form 태그 밖에 두시면 됩니다~  
이를 참고해서 form 태그를 추가해보세요!